ui/imports/shared/popups/BannerCropperModal.qml

```diff
@@ -0,0 +1,76 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Dialogs 1.3
+
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Popups 0.1
+
+import utils 1.0
+
+Item {
+    id: root
+
+    signal imageCropped(var image, var cropRect)
+
+    function chooseImageToCrop() {
+        bannerFileDialog.open()
+    }
+
+    FileDialog {
+        id: bannerFileDialog
+
+        title: qsTr("Choose an image for profile picture")
+        folder: shortcuts.pictures
+        nameFilters: [qsTr("Image files (*.jpg *.jpeg, *.jfif, *.png *.tiff *.heif)")]
+        onAccepted: {
+            if (bannerFileDialog.fileUrls.length > 0) {
+                bannerCropper.source = bannerFileDialog.fileUrls[0]
+                bannerCropperModal.open()
+            }
+        }
+    } // FileDialog
+
+    StatusModal {
+        id: bannerCropperModal
```

Comment:
This should be renamed into a more generic "Image Cropper", including the file name. Now it is used not only for banners. Don't worry can be done later.

Reply:
I will fix it)))